packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -62,17 +63,14 @@ function ChangelogTooltip<TRef extends HTMLElement>({
         return;
       }
 
-      const sendMessage = globalThis?.browser?.runtime?.sendMessage;
+      const browser = await import('webextension-polyfill-ts').then(
+        (mod) => mod.browser,
+      );
 
-      if (typeof sendMessage !== 'function') {
-        toast.displayToast(toastMessageMap.error);
-
-        return;
-      }
```

Comment:
We no longer need to support this in case of the import?
What happens if it's can't polyfill somewhere?

Reply:
Yeah makes sense to catch the "Generic" error for now.
Message can be the one we used already for when it would fail.

Reply:
Added in 0b6c2aa4be4925f1ee57f770b6987a5972662ed2